pkgs/applications/window-managers/kbdd/default.nix

```diff
@@ -17,7 +17,7 @@ stdenv.mkDerivation {
   meta = {
     description = "Simple daemon and library to make per window layout using XKB";
     homepage = "https://github.com/qnikst/kbdd";
-    license = lib.licenses.gpl3;
+    license = lib.licenses.gpl2;
```

Comment:
gpl2Plus or gpl2Only?

Reply:
Changed to gpl2Plus (since in srouce files it allows use of any later versions)